Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Undo/PrefabUndoUtils.h

```diff
@@ -17,6 +17,12 @@ namespace AzToolsFramework
     {
         namespace PrefabUndoUtils
         {
+            enum class PatchOperationType
+            {
+                Add,
+                Replace
+            };
```

Comment:
why not reuse https://github.com/o3de/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Overrides/PrefabOverrideTypes.h#L15-L20 ?